sickbeard/helpers.py

```diff
@@ -1561,3 +1561,18 @@ def canonical_name(obj, fmt='{key}:{value}', separator='|', ignore_list=frozense
     """
     guess = obj if isinstance(obj, dict) else guessit.guessit(obj)
     return str(separator.join([fmt.format(key=k, value=v) for k, v in guess.items() if k not in ignore_list]))
+
+
+def get_broken_providers():
+    """Get broken providers from cdn.pymedusa.com."""
+    url = 'https://cdn.pymedusa.com/providers/broken_providers.txt'
```

Comment:
That's what I'm saying.  Switch it to store it as json.  We also do the same for the scene exception. Will be less error prone and more future proof.